### ISSUE TITLE
Unable to find newly created user accounts

### ISSUE THREAD

**Issue by ferventcoder** [huafu/hubot-gitter2#5]
If someone that has never signed into gitter before (aka brand new user), joins gitter and then our chat room, they are not able to interact with hubot (until we restart the hubot). 

The logs indicate that the user is not found:
https://gist.github.com/ferventcoder/9dc7ea6aae9cb8db922b

However once we restart hubot, all is well again. So we have a workaround.

**Comment by huafu** [comment]
If I remember well, you need the new user to send one message in a room the bot is so that it'd be recognized.

**Comment by ferventcoder** [comment]
Do you mean send a message the bot **isn't** in? We had the same user try multiple times in a room the bot was in and it didn't work. We also have scripts that evaluate on any message, not just addressed to the bot directly

**Comment by ferventcoder** [comment]
Send a message in a room that the bot isn't in I meant...

**Comment by huafu** [comment]
No I meant send a message in a room the bot is. But apparently that did not work. That is a long time I did that adapter I don't remember exactly how I handled new users, I believe there should be a hook called and then the user automatically added but maybe I am doing this only on start. Oh yeah just remember now, yeah it is on start it scans all rooms and their users. Not sure there is a way to do else but I'll double check this when I'll have time so I let this one open
